ml-agents/mlagents/trainers/stats.py

```diff
@@ -212,7 +212,12 @@ def add_property(
 
 
 class TensorboardWriter(StatsWriter):
-    def __init__(self, base_dir: str, clear_past_data: bool = False):
+    def __init__(
+        self,
+        base_dir: str,
+        clear_past_data: bool = False,
+        hidden_keys: Optional[List[str]] = None,
+    ):
         """
         A StatsWriter that writes to a Tensorboard summary.
```

Comment:
I know this is not your fault, but could you make this method comment better and describe the parameters?

Reply:
updated the comments